### DISCUSSION TITLE
Possibility to start a custom circuit in a default state ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#985] [Q&A] [answered]
I have found that flip flops can be started with default values. Is this also possible for custom circuits ?

**Comment by hneemann** [comment, marked as answer]
It does not matter if a flipflop is in a subcircuit or not. The default value is always used.
If a self-built flipflop is used, the initial state must be defined with the reset component.

**Reply by jongena** [reply]
Again many thanks for the answer. All understood. !
Thanks once again for this nice tool !
